testproject/jijtest/build.gradle

```diff
@@ -2,10 +2,27 @@ plugins {
     id 'net.neoforged.moddev'
 }
 
+sourceSets {
+    service
+}
+
+def serviceJar = tasks.register("serviceJar", Jar) {
```

Comment:
That's nice and all, but how are you actually supposed to test this in dev? i.e. how is FML going to load it?

Reply:
> which has a manifest in META-INF/MANIFEST.MF
Your source set should probably _not_ have a manifest file placed this way -- the jar task will actually exclude it, as it's sorta required to. The manifest should be specified through the jar task itself. If moddevgradle requires folks to specify this in two different places, that's an issue

Reply:
Furthermore, running through gradle will _not_ have the jar on the classpath iirc due to secondary variants -- it'll have the classes and resources directories instead

Reply:
We need the manifest for FML to load it on the correct layer.

> Is it possible to be able to specify the FMLModType or other manifest stuff through the mods entry that groups stuff on the classpath?

Currently no. Changes to that area will happen at some point.